examples/examples_nodejs_test.go

```diff
@@ -0,0 +1,22 @@
+//go:build nodejs || all
+// +build nodejs all
+
+package examples
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/pulumi/pulumi/pkg/v3/testing/integration"
+)
+
+func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	base := getBaseOptions()
+	baseJS := base.With(integration.ProgramTestOptions{
+		Dependencies: []string{
+			"@pulumi/sumologic",
```

Comment:
We should change sumologic to foo to match the other naming conventions?

Reply:
Whoops.  That was an oversight.  Fixed.  Marking resolved.